### DISCUSSION TITLE
OpenCollective redistribution

### DISCUSSION THREAD

**Discussion by Dolu1990** [SpinalHDL/SpinalHDL#1084] [Announcements]
Hi !

I just noticed that the https://opencollective.com/spinalhdl acount grew quite a bit, and that we can now start the first redistribution to the SpinalHDL contributors :D

@Readon @kleinai @jijingg @Snoopy87 @wifasoi @numero-744 @wswslzp @andreasWallner @svancau @SomeParticles @typingArtist @piegamesde @tomverbeure @name1e5s ...

Let's me know if you want to get a share, or if you think somebody i forgot to \@ someone.

That's a first time process, let's do it right :)

**Comment by andreasWallner** [comment]
@Dolu1990 Sorry for the late reply...
TBH I don't want to put you in a tough spot about how to handle this but I'd like to opt out - it kind of does not make sense to contribute to OpenCollective (not doing it for long, but still...) and get my own money back.

**Reply by Dolu1990** [reply]
No worries ^^

**Reply by Dolu1990** [reply]
There is still the other usages of the fonds, as covering the fees for conferences / talks / travels
Also, there is the possibilities to send some SpinalHDL goodies to contributors ( Mug / T-Shirt / sticker / ... ) :)

**Comment by Dolu1990** [comment]
That's a first one <3
https://opencollective.com/spinalhdl/expenses/150518

Rawrrrrrr
